wgsl/index.bs

```diff
@@ -880,30 +880,35 @@ shader that goes beyond the specified limits.
   <thead>
     <tr><th>Limit<th>Minimum supported value
   </thead>
-    <tr><td>Maximum number of members in a [=structure=] type<td>16383
-    <tr><td>Maximum [=nesting depth=] of a [=composite=] type<td>255
+    <tr><td>Maximum number of members in a [=structure=] type<td>1024
+    <tr><td>Maximum [=nesting depth=] of a [=composite=] type<td>16
     <tr><td>Maximum nesting depth of brace-enclosed statements in a function<td>127
     <tr><td>Maximum number of [=formal parameter|parameters=] for a function<td>255
-    <tr><td>Maximum number of case selector values in a [=statement/switch=] statement<td>16383
-    <tr><td>Maximum [=byte-size=] of an [=array=] type instantiated in the
-            [=address spaces/function=] or [=address spaces/private=] address spaces
+    <tr><td>Maximum number of case selector values in a [=statement/switch=] statement<td>1024
+    <tr><td>Maximum combined [=byte-size=] of all [=variables=] instantiated in the
+            [=address spaces/private=] address space that are [=statically accessed=] by a single
+            [=shader=]
 
             For the purposes of this limit, [=bool=] has a size of 4 bytes.
-        <td>65535
-    <tr><td>Maximum [=byte-size=] of an [=array=] type instantiated in the
-            [=address spaces/workgroup=] address space.
+        <td>8192
+    <tr><td>Maximum combined [=byte-size=] of all [=variables=] instantiated in the
```

Comment:
What about private? Should that be separate or combined?

Reply:
I didn't want to overly complicate things by trying to describe the aggregation of transitively referenced private variables with function-scope declarations, if that's what you mean. IMO what is here right now is enough to be informative to shader authors whilst still being relatively easy to understand.